public/app/features/scenes/dashboard/DashboardScene.tsx

```diff
@@ -52,4 +82,74 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
     // TODO actually discard changes
     this.setState({ isEditing: false });
   };
+
+  onCloseInspectDrawer = () => {
+    locationService.partial({ inspect: null });
+  };
+
+  getPageNav(location: H.Location) {
+    let pageNav: NavModelItem = {
+      text: this.state.title,
+      url: locationUtil.getUrlForPartial(location, { viewPanel: null, inspect: null }),
+    };
+
+    if (this.state.viewPanelKey) {
+      pageNav = {
+        text: 'View panel',
```

Comment:
Think we should have a panel title in the breadcrumbs.

Reply:
Was just copying the current behavior. Not 100% if we should put panel title here:
*  as it's shown right under the breadcrumb any way, 
* And panel title is too dynamic (can include template variables) and so can change when you change template variable or even query interval